desktop/src/gui.rs

```diff
@@ -48,7 +48,10 @@ pub fn text<'a>(locale: &LanguageIdentifier, id: &'a str) -> Cow<'a, str> {
 }
 
 pub fn optional_text(locale: &LanguageIdentifier, id: &str) -> Option<String> {
-    TEXTS.lookup_single_language::<&str>(locale, id, None)
+    TEXTS
+        .lookup_single_language::<&str>(locale, id, None)
+        .inspect_err(|e| tracing::trace!("Error looking up text: {e}"))
+        .ok()
```

Comment:
Followed an `Option` -> `Result` API change here.

Opted for `trace` level to avoid spamming messages like this (for example in the language chooser combo box):
```
2024-08-19T22:19:47.966993Z TRACE ruffle_desktop::gui: Error looking up text: Couldn't retrieve message with ID `language-name`
2024-08-19T22:19:48.146270Z TRACE ruffle_desktop::gui: Error looking up text: Couldn't retrieve message with ID `language-name`
```

Reply:
We could, of course, also remove the `inspect_err` call entirely.